common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -7231,11 +7237,6 @@
       "Cannot have circular references in bean class, but got the circular reference of class <clazz>."
     ]
   },
-  "_LEGACY_ERROR_TEMP_2139" : {
```

Comment:
Could you handle both legacy conditions in the PR `_LEGACY_ERROR_TEMP_2138-9`. Seems like they are similar.